README.md

```diff
@@ -41,14 +41,16 @@ jobs:
         working-directory: checkout
 ```
 
-[.github/workflows/applanga-pull.yml](https://github.com/applanga/github-workflow-example/blob/master/.github/workflows/applanga-pull.yml) pulls new translations available from Applanga and then creates a pull request in the repo with the newly added languages or updated translation files. For this to work a **Webhook Endpoint** has to be configured for the project on Applanga dashboard to trigger the workflow. See [Configure Webhook Endpoint](#configure-webhook-endpoint) below for how to configure 
+Note that for the above workflow file the `Applanga CLI` will only get triggered when there is a push to the `master` branch as specified in the config. It is also possible to specify a different branch or add more branches as needed.
```

Comment:
Don't name it Applanga CLI people here may not really know what you mean use Applanga Push Action

Reply:
fixed